src/main/java/pl/touk/sputnik/connector/gerrit/GerritFacadeBuilder.java

```diff
@@ -42,7 +42,7 @@ public HttpClientBuilder extend(HttpClientBuilder httpClientBuilder, GerritAuthD
             }
         });
 
-        return new GerritFacade(gerritApi, gerritPatchset);
+        return new GerritFacade(gerritApi, gerritPatchset, configuration);
```

Comment:
Same above, filter need to be initialzed, I'm not sur that should be done during facade construction